### ISSUE TITLE
Add KNB as a publish node?

### ISSUE THREAD

**Issue by cboettig** [ropensci/EML#20]
Presumably we should be able to push directly to KNB as well (many benefits, including being a DataONE node...).  

May need Matt's help on getting API tools to do this...

**Comment by cboettig** [comment]
@mbjones looks like the `eml_knb` function isn't compatible with the `dataone 2.0.0` (e.g. `D1Client` not found). Clearly I need to get up to speed on the new package interface etc.

I did try a quick work-around with direct REST calls to the API.  It looks like the DEV node I was using for testing is down (502 error): https://mn-demo-5.test.dataone.org/knb/d1/mn, but checking  https://cn-dev.test.dataone.org/cn/v1/node I realized I could just switch from `5` to `6`.  I could get non-authenticated calls to work this way, but tried uploading an example file using a CIlogon certificate but I get 'permission denied' (401 error).  Maybe not worth debugging this since I should just be able to use the dataone package, but not sure why it's not working all the same.  (Example of this failing test here: https://github.com/cboettig/dataone-lite/blob/master/tests/testthat/d1_upload.R )
